src/auth/auth.styles.js

```diff
@@ -17,7 +17,11 @@ export const SignUpStyles = styled.div`
   align-items: center;
 `;
 
-export const FormStlyes = styled.div`
+export const SignUpGreetingStyles = styled.div`
+  margin-top: 72px;
+`;
```

Comment:
What happens if you put change  `justify-content: space-around;` for   `justify-content: space-evenly;` in the Form component on `auth.styles.js`. It is not that what you want to accomplish? To make the form inputs closer to each other?

Reply:
I really would prefer to talk not write, its more efficient  and prevents misunderstandings. 
I did not mean bringing margin and responsiveness together , like you understood. 
I was just asking, where are we starting and ending the styling optimization, where it is not the main part of the exercise. Sure we can make the design perfect, if we start here, than we should make it responsive as well to deliver 100% . 
It would just be cool finding then some time to really thing about the Backend thoughts we had. Like the connection between records and users